alburnum/maas/viscera/users.py

```diff
@@ -20,6 +20,12 @@ class UsersType(ObjectType):
     def __iter__(cls):
         return map(cls._object, cls._handler.read())
 
+    def create(cls, username, email, password, *, is_superuser=False):
```

Comment:
The term `is_admin` is used on the `User` class in this library, to match the terms we most often use when talking about MAAS. Change `is_superuser` here to `is_admin` or change `User.is_admin` to `is_superuser`. I prefer the former, but consistency is more important.